src/interfaces/IProvider.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import {Currency, CurrencyLibrary} from "v2-core/libraries/Currency.sol";
+import {IProviderFactory} from "./IProviderFactory.sol";
+
+interface IProvider {
+    event Claim(address indexed broker, address indexed protocolTreasury, uint256 brokerShare, uint256 protocolShare);
```

Comment:
I believe `currency` should be added here too.

Reply:
Resolved.